includes/context/RequestContext.php

```diff
@@ -208,6 +208,9 @@ public function getUser() {
 		if ( $this->user === null && $wgEnableHeliosExt ) {
 			$this->user = \Wikia\Helios\User::newFromToken( $this->getRequest() );
 			$authPath['helios'] = ($this->user !== null ? 'OK' : 'FAIL');
+			if ( $this->user !== null ) {
```

Comment:
Maybe use `instanceof` instead to enforce the object class (I assume you provide the class in method bound to a hook)?